drools-osgi/drools-karaf-itests-kjar/pom.xml

```diff
@@ -14,7 +14,6 @@
   <groupId>org.drools</groupId>
```

Comment:
While we are at it, the groupId could be removed as well. But its not strictly necessary.